qtl/interval.h

```diff
@@ -513,11 +505,12 @@ namespace qtl{
 <  <= == >=  > !=   < = >
 T  T  F  F  F  T    T F F   0 <=> 1
 F  T  T  T  F  F    F T F   0 <=> 0
-F  F  F  T  T  T    F F T   1 <=> 0
+F  F  F  T  T  T    F F T   0 <=> -1
 U  T  U  U  F  U    U U F   0 <=> (0<=x::x<=1)
-U  U  F  U  U  T    U F U   (x::x!=0) <=> (x::x==0)
-F  U  U	 T  U  U    F U U   (0<=x::x<=1) <=> 0
-U  U  U  U  U  U    U U U   (0<=x::x<=1) <=> (0<=x::x<=1)
+U  U  F  U  U  T    U F U   0 <=> (x::x!=0)
```

Comment:
consistently comparing with 0 on the left should better illustrate the relationship between intervals and partial_ordering results